src/web/command_control/client/components/RCControllerPanel.tsx

```diff
@@ -515,42 +522,46 @@ export default class RCControllerPanel extends React.Component {
 			</div>
 		)
 
-		diveControlPad = (
-			<div className='rc-dive-labels-container'>
-				<div className='rc-labels-left'>
-					{selectControlType}
-					<div className='rc-dive-info-container' >
-						<div>Max Depth:</div>
-						<input id='maxDepth' className='rc-input' type='text' value={this.props?.rcDiveParameters?.maxDepth} onChange={(evt: React.ChangeEvent<HTMLInputElement>) => this.handleTaskParamInputChange(evt)} autoComplete='off'/>
-						<div>m</div>
-
-						<div>Depth Interval:</div>
-						<input id='depthInterval' className='rc-input' type='text' value={this.props?.rcDiveParameters?.depthInterval} onChange={(evt: React.ChangeEvent<HTMLInputElement>) => this.handleTaskParamInputChange(evt)} autoComplete='off' />
-						<div>m</div>
-
-						<div>Hold Time:</div>
-						<input id='holdTime' className='rc-input' type='text' value={this.props?.rcDiveParameters?.holdTime} onChange={(evt: React.ChangeEvent<HTMLInputElement>) => this.handleTaskParamInputChange(evt)} autoComplete='off'/>
-						<div>s</div>
-
-						<div>Drift Time:</div>
-						<input id='driftTime' className='rc-input' type='text' value={this.props?.rcDiveParameters?.driftTime} onChange={(evt: React.ChangeEvent<HTMLInputElement>) => this.handleTaskParamInputChange(evt)} autoComplete='off' />
-						<div>s</div>
+		if (this.props?.rcDiveParameters !== undefined) {
```

Comment:
I don't think we need the question mark after props since rcDiveParameters is a required prop